src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -157,13 +158,20 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
 
   const auto* thread_state_slice =
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
+  auto begin_ns = std::chrono::nanoseconds(thread_state_slice->begin_timestamp_ns());
+  auto end_ns = std::chrono::nanoseconds(thread_state_slice->end_timestamp_ns());
+
   return absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s",
+      "<br/>"
```

Comment:
Nit: I think we should remove that extra break here.

Reply:
I think it's best to leave a break line between time and description, but I did delete an extra break line between the description and thread state. Please look at the new screenshot. I think it looks okay and no other changes to format are needed.